### ISSUE TITLE
Create a performance benchmarking suite for all Durable PLs and across App Service Plans

### ISSUE THREAD

**Issue by davidmrdavid** [Azure/azure-functions-durable-extension#1736]
Currently, we base our performance testing on a limited selection of PLs (C#) and app service plans. We create a more comprehensive testing matrix so we can target performance pain-points specific to user plan-and-pl combinations.

**Comment by ConnorMcMahon** [comment]
Would love to leverage test bot against this, and we could run it on some regular cadence.

**Comment by davidmrdavid** [comment]
@bachuv, is this something we could attempt?

**Comment by bachuv** [comment]
Yes, I think this could work. The test bot has options to create Consumption or Elastic Premium plans right now. I haven't worked on integrating PLs other than C#, but I think it could work because the PowerShell cmdlet that the test bot uses to create the test apps (`New-AzFunctionApp`) has a configurable runtime option.

**Comment by ConnorMcMahon** [comment]
Given the headache we are having with the v2.4.2 release, I would love to be able to run this on a nightly cadence with the latest bits so that we identify perf regressions earlier.
